packages/kbn-eslint-config/.eslintrc.js

```diff
@@ -326,6 +326,8 @@ module.exports = {
     '@kbn/imports/uniform_imports': 'error',
     '@kbn/imports/no_unused_imports': 'error',
     '@kbn/imports/no_boundary_crossing': 'error',
+    '@kbn/imports/no_group_crossing_manifests': 'warn',
+    '@kbn/imports/no_group_crossing_imports': 'warn',
```

Comment:
The plan is indeed to move to `'error'` eventually.

We discussed about the _warning_ strategy yesterday and I'm gonna push some changes to make `error` the default, and add exceptions for each of the current failures, making them `warn`, so that we can merge.